### PR TITLE
{phys}[intel/2017a] DFTB+ v1.3.1

### PR DESCRIPTION
**Pull request by migueldiascosta**
(created using `eb --new-pr`)

### CODE REVIEW

**Comment by migueldiascosta** [comment]
requires #4336

**Comment by migueldiascosta** [comment]
requires manual download after (free) registration

File: easybuild/easyconfigs/d/DFTB+/DFTB+-1.3.1-intel-2017a.eb [hunk 1]
```diff
@@ -0,0 +1,47 @@
...
+dependencies = [
+    ('arpack-ng', '3.4.0'),
+]
+
+subdir = '%dftbplus-%(version)s'
+
+prebuildopts = "sed 's/ARCH = x86_64-linux-intel/ARCH = user/' Makefile.user.template > Makefile.user && "
+prebuildopts += "cp sysmakes/make.x86_64-linux-intel sysmakes/make.user && "
+prebuildopts += "sed -i 's|LNOPT =  -static|#LNOPT =  -static|' sysmakes/make.user && "
+prebuildopts += "sed -i 's|/opt/intel/mkl/10.2.6.038/lib/em64t|$(MKLROOT)|' sysmakes/make.user && "
```
Comment (boegel) [inline easybuild/easyconfigs/d/DFTB+/DFTB+-1.3.1-intel-2017a.eb]: @migueldiascosta are these `sed` commands really needed? Can't you just override these values with arguments to `make` via `buildopts`?

```python
buildopts = 'ARCH=user LNOPT= '
```

Comment (migueldiascosta) [inline easybuild/easyconfigs/d/DFTB+/DFTB+-1.3.1-intel-2017a.eb]: you're right, simplifying

**Comment by migueldiascosta** [comment]
Test report by @migueldiascosta
**SUCCESS**
Build succeeded for 1 out of 1 (1 easyconfigs in this PR)
nscc04 - Linux RHEL 6.8, Intel(R) Xeon(R) CPU E5-2690 v3 @ 2.60GHz, Python 2.7.12
See https://gist.github.com/058f3af58aecb66fd4feadca409407a2 for a full test report.

**Comment by migueldiascosta** [comment]
Test report by @migueldiascosta
**SUCCESS**
Build succeeded for 1 out of 1 (1 easyconfigs in this PR)
grc-lin - Linux centos linux 7.1.1503, Intel(R) Xeon(R) CPU E5-2620 v2 @ 2.10GHz, Python 2.7.5
See https://gist.github.com/376fa0501ce92095c55d26a1a59a7927 for a full test report.

File: easybuild/easyconfigs/d/DFTB+/DFTB+-1.3.1-intel-2017a.eb [hunk 2]
```diff
@@ -0,0 +1,46 @@
...
+    'b821ed7406a2a4b35c0a48974e18667f',  # dftbplus-1.3.1-src.tar.xz
+    '2333f1ff826a3c4776e1e77a28f17004',  # dftbplus-1.3.1-extlib.tar.xz
+    'abaf8a2f94c74a13a0d8d19e9b022e40',  # dftbplus-1.3.1-autotest.tar.xz
+]
+
+dependencies = [
+    ('arpack-ng', '3.4.0'),
+]
+
+subdir = '%dftbplus-%(version)s'
```
Comment (boegel) [inline easybuild/easyconfigs/d/DFTB+/DFTB+-1.3.1-intel-2017a.eb]: @migueldiascosta This is unused, so please remove?

File: easybuild/easyconfigs/d/DFTB+/DFTB+-1.3.1-intel-2017a.eb [hunk 3]
```diff
@@ -0,0 +1,46 @@
...
+
+prebuildopts = "cp sysmakes/make.x86_64-linux-intel sysmakes/make.user && "
+prebuildopts += "cd prg_dftb && "
```
Comment (boegel) [inline easybuild/easyconfigs/d/DFTB+/DFTB+-1.3.1-intel-2017a.eb]: collapse to a single line?

**Comment by boegel** [comment]
Test report by @boegel
**SUCCESS**
Build succeeded for 1 out of 1 (1 easyconfigs in this PR)
node2006.delcatty.os - Linux centos linux 7.3.1611, Intel(R) Xeon(R) CPU E5-2670 0 @ 2.60GHz, Python 2.7.5
See https://gist.github.com/8f6169db4f13b368a4f06f7f89d188a6 for a full test report.

**Comment by boegel** [comment]
lgtm

**Comment by boegel** [comment]
Test report by @boegel
**SUCCESS**
Build succeeded for 2 out of 2 (1 easyconfigs in this PR)
node2410.golett.os - Linux centos linux 7.3.1611, Intel(R) Xeon(R) CPU E5-2680 v3 @ 2.50GHz, Python 2.7.5
See https://gist.github.com/b22b0e4522470b0c9ceee3e8d65dd155 for a full test report.

**Comment by boegel** [comment]
Going in, thanks @migueldiascosta!

**Pull request merged**